src/features/callAssignments/hooks/useCallAssignmentStats.ts

```diff
@@ -126,8 +126,10 @@ export default function useCallAssignmentStats(
   };
 
   return {
-    ...getStats(),
+    data: getStats().data,
+    error: getStats().error,
```

Comment:
For me all this calls to `getStats()` look unnecessary. Is it possible to save the return of the function in a variable and work with that in the return? Like stats.data, stats.error or event just return the future object?